public/index.html

```diff
@@ -2,39 +2,25 @@
 <html>
   <head>
     <meta charset="UTF-8" />
-    <style>
-      body {
-        margin: 0;
-        background: rgb(39, 0, 71);
-        height: 100vh;
-        display: flex;
-        flex-direction: column;
-        justify-content: center;
-        align-items: center;
-      }
-
-      canvas {
-        background-color: white;
-      }
-      button {
-        padding: 10px 20px;
-        font-size: x-large;
-        width: 80%;
-        height: 20%;
-        background-color: rgb(0, 128, 128);
-        color: rgb(218, 165, 32);
-        border: none;
-        cursor: pointer;
-        font-weight: bold;
-      }
-
-      .title {
-        color: white;
-      }
-    </style>
+    <link rel="stylesheet" href="style.css" />
+    <title>Thetawave</title>
+    <link rel="icon" href="favicon.png" type="image/x-icon" />
   </head>
 
   <body>
-    <button id="buttonToTriggerGame" type="button" onClick="location.href='/thetawave'">Click to Play Thetawave!</button>
+    <div class="screen scanlines">
+      <div class="flexbox hcenter screen-content">
+        <div id="logo-container" class="hcenter">
+          <img src="logo.gif" width="800"  alt="logo"/>
```

Comment:
```suggestion
          <img src="https://assets.thetawave.metalmancy.tech/promo/thetawave_logo_animated.gif" width="800"  alt="logo"/>
```
I put this thing in S3, since we have used it a couple of times (probably better than google drive). Depending on how i set up CORS, we may need to do a little redirect in "dev_netlify.toml" and "prod_netlify.toml" to route "/logo.gif" to "https://assets.thetawave.metalmancy.tech/promo/thetawave_logo_animated.gif"